src/main/java/gnu/io/Zystem.java

```diff
@@ -225,7 +225,7 @@ public boolean report( String s)
 		}
 		else if ( mode == PRINT_MODE )
 		{
-			log.info( s );
+			log.debug( s );
```

Comment:
This whole class exists to facilitate logging, but is not documented, and AFAIK, not used. That you've made changes here is fine, but the next step in refactoring logging is probably to remove this class altogether, so don't worry too much about any other changes to it.